images/capi/packer/ova/ubuntu-2404-efi.json

```
@@ -0,0 +1,18 @@
+{
+  "boot_command_prefix": "c<wait>linux /casper/vmlinuz ipv6.disable={{ user `boot_disable_ipv6` }} --- autoinstall ds='nocloud-net;s=http://{{ .HTTPIP }}:{{ .HTTPPort }}/24.04.efi/'<enter><wait>initrd /casper/initrd<enter><wait>boot<enter>",
```

```suggestion
  "boot_command_prefix": "c<wait>linux /casper/vmlinuz ipv6.disable={{ user `boot_disable_ipv6` }} --- autoinstall ds='nocloud'<enter><wait>initrd /casper/initrd<enter><wait>boot<enter>",
```
Maybe this helps (also at the other).

We're trying to read the data from a floppy, not via http. The VM is not able to reach the processes we are running here.

---

Oh that is interesting. Worth trying this out